ffi/inkview_h.lua

```diff
@@ -1159,6 +1159,9 @@ void AddLastOpen(const char *);
 void AddScrolledArea(struct irect_s *, int);
 void AddTranslation(const char *, const char *);
 int AdjustDirectionKeys(int);
+void adjustAreaDefault(unsigned char* data, int scanline, int width, int height);
+void adjustAreaWithVibrance(unsigned char* data, int scanline, int width, int height, float vibrance);
+void adjustAreaWithLabColorsSaturation(unsigned char* data, int scanline, int width, int height, double koef);
```

Comment:
Please also add this to https://github.com/koreader/koreader-base/blob/c1d97b47abeca6eccd0c136e31749302cc84212f/ffi-cdecl/inkview_decl.c

Reply:
Added just now